src/Composer/ScriptHandler.php

```diff
@@ -71,18 +71,23 @@ public static function generateRandomSecret(Event $event)
      *
      * @throws \RuntimeException
      */
-    private static function executeCommand($cmd, Event $event)
+    protected static function executeCommand($cmd, Event $event)
     {
         $phpFinder = new PhpExecutableFinder();
 
         if (false === ($phpPath = $phpFinder->find())) {
             throw new \RuntimeException('The php executable could not be found.');
         }
 
+        if (false === file_exists($consolePath = static::getConsoleScript($event))) {
+            throw new \RuntimeException('The Symfony console could not be found.');
+        }
+
         $process = new Process(
             sprintf(
-                '%s bin/console%s %s%s',
```

Comment:
Haven't we already changed this to read the bin directory from the composer configuration?